frontend/hooks/useFileStorage.js

```diff
@@ -10,55 +10,139 @@ const INITIAL_DATA = {
   totalItems: 0,
 };
 
+const TIMEOUT_DURATION = 5000;
+
 export default function useFileStorage(
   fileStorageId,
-  path = '',
-  sortKey = null,
-  sortOrder = null,
+  path = '/',
+  sortKey = 'updatedAt',
+  sortOrder = 'desc',
   page = 1,
 ) {
   const previousData = useRef();
   const queryClient = useQueryClient();
 
-  const { data, isLoading, isFetching, isPending, isPlaceholderData, error } = useQuery({
-    queryKey: ['fileStorage', fileStorageId, path, sortKey, sortOrder, page],
-    queryFn: () =>
-      federalist.fetchPublicFiles(fileStorageId, path, sortKey, sortOrder, page),
-    refetchInterval: REFETCH_INTERVAL,
-    refetchIntervalInBackground: false,
-    enabled: !!fileStorageId,
-    keepPreviousData: true,
-    staleTime: 2000,
-    placeholderData: previousData.current || INITIAL_DATA,
-    onError: (err) => {
-      // using an empty string so that we don't end up with "undefined" at the end
-      throw new Error('Failed to fetch public files ' + (err?.message || ''));
-    },
-  });
+  const [uploadError, setUploadError] = useState(null);
```

Comment:
@apburnes i know it looks like a lot, but this handles dismissing the alerts (both error and success) after a set duration. I havent added a dismiss click yet.